package.json

```
@@ -9,7 +9,9 @@
     "build": "pnpm -r build",
     "check": "pnpm -r check",
     "lint": "pnpm -r lint",
-    "format": "pnpm -r format"
+    "format": "pnpm -r format",
+    "version": "changeset version && pnpm -r generate:version && git add -a",
```

```suggestion
    "changeset:version": "changeset version && pnpm -r generate:version && git add -A",
```

`pnpm version` prints the version, you'd have to remember to always call `pnpm run version`. to avoid this conflict i propose renaming this script. 

and the flag is --all or -A, sorry, i mistyped that in discord :E

---

ah, yeah. good catches!